tests/helper/helper.go

```diff
@@ -913,3 +924,224 @@ func CheckKubectlGetResult(t *testing.T, kind string, name string, namespace str
 	unqoutedOutput := strings.ReplaceAll(string(output), "\"", "")
 	assert.Equal(t, expected, unqoutedOutput)
 }
+
+// KedaConsistently checks if the provided conditionFunc consistently returns true
+// (and no error) for the entire duration specified by the context's deadline.
+// It polls the conditionFunc at the given interval.
+func KedaConsistently(ctx context.Context, conditionFunc wait.ConditionWithContextFunc, interval time.Duration) error {
```

Comment:
given keda already vendors gomega, could we use the testing matchers provided by that library? In this case it seems like with a little refactor, we could use `Consistently().Should()` pattern
https://github.com/kedacore/keda/blob/dcfae290507c3b5936abe7dc9c75969ceaa785f9/vendor/github.com/onsi/gomega/gomega_dsl.go#L427

Reply:
Cool, I took a really quick look to see if other tests were using gomega already, but didn't see anything. If you guys already vendor it, then it makes sense for me to refactor, thanks! :+1: